dev/source/docs/mavproxy-developer-gcs.rst

```diff
@@ -14,10 +14,4 @@ multiple other ground station software on other devices.
 MAVProxy is commonly used by developers (especially with SITL) :ref:`for testing new builds <using-sitl-for-ardupilot-testing>`.
 
 The :ref:`MAVProxy Documentation<mavproxy:home>` contains detailed
-information on how to use the tool. This section contains complementary
-articles and tutorials:
-
-.. toctree::
-    :maxdepth: 1
-
-    MAVProxy on Windows 7 <mavproxy-on-windows-7>
```

Comment:
PS also causes a build error its referenced from other pages

Reply:
PPS  sorry if this sounded perturbed or angry, its not!....just was surprised....also, we usually dont delete pages, we archive them and add to archived list main page so they dont cretae TOC errors....

Reply:
Maybe moving to archive is good but that would be true only if we still have a release available for windows 7 .

Reply:
I thought archiving was to allow someone that already had it installed or used, and is coming back after some time and wants info on it...or allow a support question from them to be answered on an older installation.....its not to allow them to install now...otherwise, I do not know what purpose archiving serves at all...

Reply:
you are part right ... the question on discuss show for example that APM2.5 boards are still used and that people still use the archived pages ... I am pretty sure that it will be the same for windows 7 ... some people don't want to use windows10 and will still wanting to use mavproxy on windows 7 (or mission planner on windows XP ... )